tracee-ebpf/go.mod

```diff
@@ -2,15 +2,13 @@ module github.com/aquasecurity/tracee/tracee-ebpf
 
 go 1.16
 
-replace github.com/aquasecurity/tracee/tracee-ebpf/external => ./external/
-
```

Comment:
Why removing this?

Reply:
This was done automatically by GoLand, I'll check why that happened. I see it was already put back.